### PR TITLE
128 added falcon dashboard tuner

### PR DESCRIPTION
**Pull request by Jeffthedoor**
has support for PIDSV

### CODE REVIEW

File: tuning/FalconTuner.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+    private double kP;
+    private double kI;
+    private double kD;
+    private double kS;
+    private double kV;
+
+    private static int tunerNumber = 0;
+
+    public FalconTuner(String name, Slot0Configs PIDConfigs, TalonFX motor) {
```
Comment (WindowsVistaisCool) [inline tuning/FalconTuner.java:30]: motor is never being set (or used)

File: tuning/FalconTuner.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            kPTuner = tab.add(name + "kP", PIDConfigs.kP)
+                    .withPosition(3, tunerNumber).getEntry();
+            kITuner = tab.add(name + "kI", PIDConfigs.kI)
+                    .withPosition(4, tunerNumber).getEntry();
+            kDTuner = tab.add(name + "kD", PIDConfigs.kD)
+                    .withPosition(5, tunerNumber).getEntry();
+            kSTuner = tab.add(name + "kS", PIDConfigs.kS)
+                    .withPosition(6, tunerNumber).getEntry();
+            kVTuner = tab.add(name + "kV", PIDConfigs.kV)
+                    .withPosition(4, tunerNumber).getEntry();
```
Comment (WindowsVistaisCool) [inline tuning/FalconTuner.java:58]: is that supposed to be 4?

**Review by WindowsVistaisCool — APPROVED** [review]
looks okay otherwise

**Comment by Jeffthedoor** [comment]
@frc-862/reviewers please review again :)

File: tuning/SlotConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,228 @@
+package frc.thunder.tuning;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.signals.GravityTypeValue;
+ 
+// This class is used to set the PIDF gains for a TalonFX Config for a specified slot number. it is primarily for backend use.
+public class SlotConfiguration {
+    private int slot;
```
Comment (WindowsVistaisCool) [inline tuning/SlotConfiguration.java:8]: could be final

**Review by WindowsVistaisCool — APPROVED** [review]
this pr is xi jing ping approved 🔥🔥🔥✅✅✅

**Pull request merged**